Server/Plugins/APIDump/Classes/Plugins.lua

```diff
@@ -22,10 +22,10 @@ return
 
 	cPluginLua =
 	{
-		Desc = "",
+		Desc = "(<b>OBSOLETE</b>) This class is no longer useful in the API and will be removed as soon as all core plugins are migrated away from it. The {{cPlugin}} class serves as the main plugin instance's interface.",
```

Comment:
I can't really remember the exact details of what you're referring to. I guess my concern was that once a plugin registered a web tab, the handler for the web tab would need to be valid for all webadmin calls, which happen in different threads. This would be solved by `cCallback` being properly thread-locked and staying valid even after its original `cLuaState` is gone.